mautic/mautic
# Fix contact value campaign condition with company select field

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | features
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | Fixes #10149 




#### Description:
Contact field value campaign condition failed If you use custom company select field ad include/exclude operator. 
This PR fixed it.



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create custom company select field
3. Create campaign with contact field value condition
4. Select custom company select field and include operator.
5. Run campaign 
6. Should return error

`Type error: Argument 1 passed to Mautic\LeadBundle\Helper\CustomFieldHelper::fieldValueTransfomer() must be of the type array, null given`

7. After patch works properly
